### PR TITLE
Add DIA to oracles section

### PR DESCRIPTION
**Pull request by mikelberlin**
### Description

Please explain the changes this PR addresses here.

### Checklist

- [ ] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [ ] If pages have been moved around, I have run the `move-pages.py` script to move the pages and update the image paths on the chinese repo
    - [ ] After the script has been run, I have created an additional PR in `moonbeam-docs-cn`
- [ ] If images have been added, I have run the `compress-images.py` script to compress the images.
- [ ] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables
- [ ] If this page requires a disclaimer, I have added one

### Corresponding PRs

Please link to any corresponding PRs here.

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [ ] No additional PRs are required after the translations are done

#### Items to be Updated

Please list any of the items that will need to be added or deleted after the translations are done here.

### CODE REVIEW

File: builders/integrations/oracles/dia.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+
+[DIA/USD](https://diadata.org/app/price/asset/Ethereum/0x84cA8bc7997272c7CfB4D0Cd3D55cd942B3c9419/), [BTC/USD](https://diadata.org/app/price/asset/Bitcoin/0x0000000000000000000000000000000000000000/), [USDC/USD](https://diadata.org/app/price/asset/Ethereum/0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48/)
+
+## Supported token API endpoints
+
+DIA also supports API and GraphQL endpoints to return cryptocurrency price data. You can [visit the DIA Documentation](https://docs.diadata.org/products/token-price-feeds/access-api-endpoints) to see all API endpoints.
+
+# 🎨 NFT Floor Price Feeds
+
+DIA NFT floor price feeds provide smart contracts with real-time price information of [18,000+ NFT collections](https://diadata.org/app/floor-price), sourced on-chain with 100% transparency from [multiple, cross-chain NFT marketplaces](https://diadata.org/app/source/nft).
```
Comment (eshaben) [inline builders/integrations/oracles/dia.md:57]: Do you currently have any NFT collections from Moonbeam or Moonriver? I searched a few and nothing came up, but yeah just curious 🙂

File: builders/integrations/oracles/dia.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+DIA has deployed the following demo oracles for the Moonbeam community. It provides a limited selection of cryptocurrency price feeds with predefined configuration settings.
+
+> ⚠️ NOTE: DIA demo oracles are not intended for use in production environments. Developers can request a dedicated, production-ready oracle with custom price feeds and configuration settings. Start the request process: [Request a Custom Oracle | DIA Documentation](https://docs.diadata.org/introduction/intro-to-dia-oracles/request-an-oracle)
+
+### Demo Oracle Smart Contracts
+
+| Network        | Contract address      
+| -------------- | -------------------------------------------------------------------------------------------------------------------------------------- |
+| Moonbeam Alpha  | [`0xe23d8713aa3a0a2c102af772d2467064821b8d46`](https://moonbase.moonscan.io/address/0xe23d8713aa3a0a2c102af772d2467064821b8d46)        |
+| Moonriver  | [`0x11f74b94afb5968119c98ea277a2b73208bb39ab`](https://moonriver.moonscan.io/address/0x11f74b94afb5968119c98ea277a2b73208bb39ab)        |
```
Comment (eshaben) [inline builders/integrations/oracles/dia.md:45]: Is there one for Moonbeam?

File: builders/integrations/oracles/dia.md (hunk 1 above)
Comment (eshaben) [inline builders/integrations/oracles/dia.md:57]: Do you currently have any NFT collections from Moonbeam or Moonriver? I searched a few and nothing came up, but yeah just curious 🙂

File: builders/integrations/oracles/dia.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+{
+	"round": 1597683,
+	"randomness": "24138936fcbf7fc3951c928158be6998cee3af622142d0790333608d17a5c5f6",
+	"signature": "8c04905c0adf34f1fb007915d9ccc7d07b97305fc63952726f9367c87f36ab687c5e190c151f6ac4d760a9e009fc54230adb8513885449d649a229bc727be9ff347bdbce1c609cebf993b6ae57133fbcf23f96b15dbd3510cb5f2ade6b30b647",
+	"previous_signature": "ada42197a2db89866da4c44348f77f7868e41e961ec32e636b912d43c625386afae9e54944ac573047dbd227ee495b52059586c8d8cd0edfe18cc15ca0666a66651da1d62b12af2d0fac19735bed9298690a593571965c3ad7c7b11947e76ec0"
+}
+```
+
+
+The DIA randomness smart contract is structured as follows
```
Comment (eshaben) [inline builders/integrations/oracles/dia.md:80]: I checked out your docs, looks like you have a demo randomness contract deployed to Moonbase Alpha ([`0x48d351ab7f8646239bbade95c3cc6de3ef4a6cec`](https://moonbase.moonscan.io/address/0x48d351ab7f8646239bbade95c3cc6de3ef4a6cec)). Do you by chance have one for Moonriver or Moonbeam?

**Review by eshaben — COMMENTED** [review]
@mikelberlin thanks for submitting this! 💜 I just have a couple questions!

File: builders/integrations/oracles/dia.md (hunk 1 above)
Comment (mikelberlin) [reply builders/integrations/oracles/dia.md:57]: Currently, we do not have any NFT markets integrated, but that is something that can be done on demand.

File: builders/integrations/oracles/dia.md (hunk 2 above)
Comment (mikelberlin) [reply builders/integrations/oracles/dia.md:45]: Unfortunately, we currently don't have a demo oracle for Moonbeam, as these are just for testing. DIA deploys custom oracles for every dApp and use case, to create the best data solution. That's why we believe it's sufficient to have a demo oracle on Moonbase Alpha. But that is also something we could set up if required!

File: builders/integrations/oracles/dia.md (hunk 3 above)
Comment (mikelberlin) [reply builders/integrations/oracles/dia.md:80]: We don't, for the same reason as before. But again, something that could be set up if required :) Let me know!

**Pull request merged**